src/sybil/query_spec.go

```diff
@@ -80,7 +80,7 @@ type Aggregation struct {
 	opID     int
 	Name     string
 	nameID   int16
-	HistType string
+	HistType HistogramType
```

Comment:
you are breaking my query cache here. i need a sybil command to clean caches (or expire cached results after X time period)

Reply:
I can back out that type change.

Reply:
As discussed I think I'll be suggesting a alternate method for cache key generation that would have the benefit of being stable in the case of this type of change.